### ISSUE TITLE
Loot issues

### ISSUE THREAD

**Issue by Ravyd** [elysium-project/server#1562]
Loot Issues.

Loot in groups has to constantly be babied to be picked up. This means that for any given mob, for one player the greys and whites may drop, and for the rest the money will be lootable. Then we add greens to the equation. So,

a. one guy loots the mob, gets his runecloth (having trashloot rights).

b.That corpse then shows loot to the rest of the group as 10 silver plus a green which is then rolled on.

c. Someone loots the money which is split, leaving the green.

d. Then the green is rolled on. After the green is rolled on and loot is given, the corpse STILL shows as having loot to the entire group, forcing someone to loot it again to find nothing and clearing the loot indicator.

This causes people having to CONSTANTLY reloot corpses over and over and over again. It is extremely annoying. Each corpse requires 3 people looting it, and then you add skinning to the equation. It is infuriating and causes people to simply give up on looting things like black dragons which skinners need to have cleared to loot. All because someone is just sick of bending down to pick up some 6 silver that the trashloot holder should have cleared already. In vanilla wow when loot was distributed by roll corpses automatically cleared themselvesof sparkles. This could be the thing bugging the skins as well. People know the corpse is empty so they skip it, but there's still a loot indicator there, prohibiting skinning. 

I constantly find just 10 silver of loot sitting left on a mob in groups. I don't remember that EVER happening in vanilla unless I was solo and the mob only dropped 10 silver and NOTHING ELSE. Something about coinage is buggy. Coinage should be cleared by the first person who loots it, PERIOD. Unless that person specifically chose not to loot the money which happens maybe 1 out of every 100 times. 99% of loots are shift clicks. Shift clicks should clear coinage and trashloot leaving only greens, rares and epics which should clear themselves after the roll. Coinage and trashloot should only be lootable by that person unless they chose to loot it, take nothing, and move on at which point it is FFA.

 

On that note, there is no chat log indicator of loot being delivered to the winner of the roll. In vanilla wow after "X won the roll for Y" there would be a follow up chat log of "X receives loot: Y".

There is an additional problem; if the winner's bags are full loot privilege is not reserved for them, anyone can at the point go loot the item freely.

 

and finally, another thing regarding loot. Scourge stones and the like are distributed on a free for all basis. Meaning, if we're both wearing Argent Dawn trinkets and you're a rogue and im a healer, you can freely loot every single scourgestone in reach, and if they're all gone before I get there (which happens 90% of the time with melee sucking up every possible stone before healers and ranged can react) there's nothing I can do about it. Melee on an average scholo or strat run probably get at the minimum 75% of the stones dropped in the instance. The only stones that are actually fair are boss stones that are rolled on. This is EXTREMELY unfair to ranged, and especially healers. As a mage, I frequently need to be drinking after pulls not worrying about how I'm gonna blink over and loot corpses before the vacuum cleaner of a rogue does. If I'm drinking I can't be looting, so I have to decide between stones or being a cohesive member of my team.

 

I can't remember, but I highly doubt token loot (felwood corruptor's shards or whatever, scourestones, etc.) was a free for all system back in vanilla wow. It is inherently unfair to half of the group.

 

Thank you for all your hard work, I'll leave a copy of this in your bug reporting. After my mind started jogging a little bit I remembered more and more.

 

 

Please fix the loot though it is exceedingly annoying. Out of all the bugs I'd say the loot issue completely brings you out of the moment and makes you aware that this is not an official server.

**Comment by thepparker** [comment]
The only situation I'm aware of that causes the lootable icon to remain on a mob after a roll distribution is when someone still has a quest item sitting on it. 

Coins should always be FFA. Only the person who is allocated loot on a mob may take the round robin greys/whites until they open the loot window, at which point anyone may loot the items. If the person who is allocated the round-robin loots, any items at or above the loot threshold are rolled on. I know of no situation where a green or above (depends on threshold) is not rolled on the first loot and requires multiple attempts for the roll to pop up.

I'm fairly sure that if your bags are full and you win a roll, you forfeit the allocation and anyone can loot it. You'd have to find a video or something showing otherwise. Loot rolls and allocation are shown in the chat window, you likely either have an addon that is hiding it, or you were out of range of the receiver.

No comment on Scourgestones and other spell/aura dependent lootables, would need some evidence either way.

**Comment by Ravyd** [comment]
Okay, then you agree the loot icon should disappear once the item is rolled on and distributed. Bugged.

If any item above the threshold is on the corpse, anyone can open loot on the creature. So that's also bugged. Regardless of if there is loot above the threshold or not, only that person can open the loot window. This INCLUDES money.

You complete misread my post. Nowhere did I say that multiple attempts were required for the roll to pop up.

And finally, no. At no point in wow's history did you lose your item because your bags were full. Even if they were that item was reserved to you until you looted it. Which is why the last part of my post saying that "X receives Y" was there to show if the person got it or not

**Comment by thepparker** [comment]
> If any item above the threshold is on the corpse, anyone can open loot on the creature. So that's also bugged. Regardless of if there is loot above the threshold or not, only that person can open the loot window. This INCLUDES money.

Incorrect, otherwise no one but the current round robin player could loot bosses to trigger a loot roll. As you can clearly see in any video from Vanilla, if there are items that would be rolled on, every player can loot the corpse.

> You complete misread my post. Nowhere did I say that multiple attempts were required for the roll to pop up.

> > b.That corpse then shows loot to the rest of the group as 10 silver plus a green which is then rolled on.

> > c. Someone loots the money which is split, leaving the green.

> > d. Then the green is rolled on. 

> Which is why the last part of my post saying that "X receives Y" was there to show if the person got it or not

As I stated, the message DOES appear. It is shown when a player receives the item and it is placed in their bags. If their bags are full, the item remains on the corpse until looted. Maybe it should be restricted to the person who won the item, maybe not. Your memory is not a valid source for the correct behaviour.

**Comment by Phatcat** [comment]
> Only the person who is allocated loot on a mob may take the round robin greys/whites until they open the loot window, at which point anyone may loot the items

No; until they CLOSE the loot window, not when they open it.

> I'm fairly sure that if your bags are full and you win a roll, you forfeit the allocation and anyone can loot it.

What? No. Just no. There's a sort of cooldown timer on rolled for loot, if it hasn't been picked up after a few minutes, then it becomes ffa, not immediately if someone has full bags...

> Incorrect, otherwise no one but the current round robin player could loot bosses to trigger a loot roll. As you can clearly see in any video from Vanilla, if there are items that would be rolled on, every player can loot the corpse.

That's... That's what he said. He said that if there is any loot above or at the threshold, anyone should be able to trigger the roll or if there's money on the corpse, be able to loot it. Currently only the loot recipient (according to round robin) is allowed that atm. And you say incorrect?

> Maybe it should be restricted to the person who won the item, maybe not.

No, you're right, maybe the person who won the item shouldn't have... won the item... if he didn't check his bags first... Does this sound like something Blizzard would do? How many bug reports you think they would get about people losing their items because of full bags and someone else taking the item THEY won? - Think for a second, man... some things only take a bit of common sense -.-

> Your memory is not a valid source for the correct behaviour.

And neither is yours. Where are any of your sources countering what he is saying? ...

**Comment by Rockee** [comment]
I've just ran several instances on Elysium, and it was just awkward having to ask people to trade green items that i've won when i had full bags.
Hoping for more insight on what's causing this problem

**Comment by jsb** [comment]
I think this issue should have higher priority. Group looting is a core gameplay mechanic and the faulty behavior negatively affects almost all players (loot quests, Scourgestones).

**Comment by Cabro** [comment]
This is getting fixed soon™ on the live servers, as they have already fixed it on their dev realm. #2947
